src/Common/Admin/Conditional_Content/Datetime_Conditional_Abstract.php

```diff
@@ -1,73 +1,78 @@
 <?php
-namespace Tribe\Admin\Conditional_Content;
+
+namespace TEC\Common\Admin\Conditional_Content;
 
 use Tribe__Date_Utils as Dates;
+use Tribe__Template as Template;
+use Tribe\Utils\Date_I18n;
 
 /**
  * Abstract class for conditional content.
  *
- * @since 4.14.7
+ * @since TBD
  */
 abstract class Datetime_Conditional_Abstract {
 	/**
 	 * Item slug.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 */
-	protected $slug = '';
+	protected string $slug = '';
 
 	/**
 	 * Start date.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 */
-	protected $start_date;
+	protected string $start_date;
 
 	/**
 	 * Start time.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 */
-	protected $start_time;
+	protected string $start_time;
 
 	/**
 	 * End date.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 */
-	protected $end_date;
+	protected string $end_date;
 
 	/**
 	 * End time.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 */
-	protected $end_time;
+	protected string $end_time;
 
 	/**
 	 * Stores the instance of the template engine that we will use for rendering the page.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
 	 *
-	 * @var \Tribe__Template
+	 * @var Template
 	 */
-	protected $template;
+	protected Template $template;
 
 	/**
 	 * Register actions and filters.
 	 *
-	 * @since 4.14.7
+	 * @since TBD
+	 *
 	 * @return void
 	 */
-	abstract function hook();
+	abstract public function hook(): void;
 
 	/**
 	 * Unix datetime for content start.
 	 *
-	 * @since 4.14.7
-	 * @return \Tribe\Utils\Date_I18n - Date Object
+	 * @since TBD
+	 *
+	 * @return ?Date_I18n - Date Object
 	 */
-	protected function get_start_time() {
+	protected function get_start_time(): ?Date_I18n {
 		$date = Dates::build_date_object( $this->start_date, 'UTC' );
```

Comment:
This returns `now` on fail (third param default) perhaps we should not do that and instead have it return false.

Then the lower return of null (and checks on it) could be for boolean false.